Tensile/KernelWriterAssembly.py

```diff
@@ -9055,41 +9055,40 @@ def localSplitULocalRead(self, kernel):
   def localSplitUReduction(self, kernel):
     kStr = ""
 
-    is_non_hpa_fp16 = kernel["ProblemType"]["DataType"].isHalf() and (not kernel["ProblemType"]["HighPrecisionAccumulate"])
-    elementStep = 2 if is_non_hpa_fp16 else 1
-    regsPerElem = kernel["ProblemType"]["DataType"].numRegisters()
+    is_fp16 = kernel["ProblemType"]["ComputeDataType"].isHalf()
+    elementStep = 2 if is_fp16 else 1
+    regsPerElem = kernel["ProblemType"]["ComputeDataType"].numRegisters()
 
     for r in range(1, kernel["LocalSplitU"]):
       for i in range(0, kernel["NumGlobalWriteVectorsPerThread"]):
         for s in range(0, kernel["GlobalWriteVectorWidth"], elementStep):
           cIdx = int((s + i * kernel["GlobalWriteVectorWidth"]) * regsPerElem)
           regIdx = int((s + i * kernel["GlobalWriteVectorWidth"] + r * kernel["GlobalWriteVectorWidth"] * kernel["NumGlobalWriteVectorsPerThread"]) * regsPerElem)
 
-          if is_non_hpa_fp16:
+          if is_fp16:
             kStr += inst("v_pk_add_f16", vgpr("ValuC+%u"%cIdx), vgpr("ValuC+%u" % regIdx), vgpr("ValuC+%u"%cIdx), \
                          "c[%u] += c[%u]"%(cIdx, regIdx) )
-          elif kernel["ProblemType"]["DataType"].isInt8x4():
+          elif kernel["ProblemType"]["ComputeDataType"].isInt8x4():
```

Comment:
This is not correct.
ComuputeDataType should be isInt32 (as I showed in my previous comment).